lib/components/Images/image.css

```diff
@@ -0,0 +1,33 @@
+.image-component {
+    display: flex;
+    flex-flow: column;
+    gap: 1rem;
+}
+
+.image-component iframe{
+    height: 8rem;
+}
+
+.container-img {
+    padding: 1rem;
+    box-shadow: var(--shadow-color) 0px 3px 8px;
+    border-radius: 1rem;
+}
+
+.responsive-img {
+    width: 100%;
+    height: 30rem;
+}
+
+.round-image-container {
```

Comment:
TIP: You could use 

display: grid; 
place-content: center

shorter method to center stuff

Reply:
I didn't knew about this, thanks for the advice